conftest.py

```diff
@@ -88,6 +88,13 @@ def db(engine, sqlalchemy_connect_url):
     Base.metadata.create_all(engine)
 
 
+@pytest.fixture(scope="session")
```

Comment:
Fixture to avoid django trying to recreate a DB and instead pointing to the preexisting DB